pyproject.toml

```diff
@@ -22,7 +22,8 @@ classifiers = [
 dependencies = [
     "requests==2.31.0",
     "psutil==5.9.8",
-    "packaging==23.2"
+    "packaging==23.2",
+    "termicolor==1.0.0"
```

Comment:
Just checking, term*i*color is correct? It looks like termcolor everywhere else

Reply:
nope you are definitely right